--- jpgraph-2.3.4/src/jpg-config.inc.php~	2009-06-15 17:35:46.000000000 +0300
+++ jpgraph-2.3.4/src/jpg-config.inc.php	2009-06-15 17:36:32.632029137 +0300
@@ -36,7 +36,7 @@
 //
 //------------------------------------------------------------------------
 // define('CACHE_DIR','/tmp/jpgraph_cache/');
-// define('TTF_DIR','/usr/share/fonts/truetype/');
+define('TTF_DIR','/usr/share/fonts/TTF/');
 // define('MBTTF_DIR','/usr/share/fonts/truetype/');
 
 //-------------------------------------------------------------------------
@@ -123,7 +123,7 @@
 // Please note that the Apache user must be a member of the
 // specified group since otherwise it is impossible for Apache
 // to set the specified group.
-define('CACHE_FILE_GROUP','www');
+define('CACHE_FILE_GROUP','http');
 
 // What permissions should the cached file have
 // (Set to '' will give the default persmissions for the 'PHP-user')

